article/Future-work/future-work.tex

```diff
@@ -1 +1,14 @@
-\section{Future work}
\ No newline at end of file
+\section{Future work}
+We have several ideas for future work.
+There are obviously other methods to calculate the weighted summation layer combinations that could be investigated.
+Additionally these layer combinations could be based on information about the dataset, so that it is not necessary to run each layer individually.
```

Comment:
```suggestion
Additionally, the weight of each layer in the weighted summation could be based on patterns in the dataset, so that it is not necessary to run each layer individually.
```